torch/nn/modules/instancenorm.py

```diff
@@ -6,6 +6,7 @@ class _InstanceNorm(_BatchNorm):
     def __init__(self, num_features, eps=1e-5, momentum=0.1, affine=False):
         super(_InstanceNorm, self).__init__(
             num_features, eps, momentum, affine)
+        self.use_running_stats = False
```

Comment:
This property name collides with the method of the same name. We should rename this, perhaps to `self._use_running_stats`.

Reply:
omg, you are absolutely correct. I'll send out a fix!